apps/s_client.c

```diff
@@ -912,7 +907,7 @@ int s_client_main(int argc, char **argv)
             no_prot_opt++;
         if (prot_opt == 1 && no_prot_opt) {
             BIO_printf(bio_err, "Cannot supply both a protocol flag and "
-                                "\"-no_<prot>\"\n");
+                       "\"-no_<prot>\"\n");
```

Comment:
Not sure what our style actually says about this. I kind of preferred the previous version.